.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,211 @@
+name: Test Windows
+
+on: [push, pull_request]
+
+jobs:
+  build:
+  
+    runs-on: windows-2016
+    strategy:
+      matrix:
+        python-version:
+          - "3.5.4"
+          - "3.6.8"
+          - "3.7.4"
```

Comment:
Do we need to specify the full 3.x.y version, or can we use 3.x to have the latest one available, like for Linux and macOS?

https://github.com/python-pillow/Pillow/pull/4081/files#diff-fe8421955fd596131bb6f1b78984b2fbR17

Reply:
Fixed on my branch, thanks!